dcp_diag/component_agents/upload_entities.py

```diff
@@ -77,7 +114,7 @@ class DbChecksum(Base):
     file = relationship("DbFile", back_populates='checksums')
 
     def __str__(self, prefix="", verbose=False):
-        output = colored(f"{prefix}Checksum {self.id}\n", 'blue') + \
+        output = colored(f"{prefix}Checksum {self.id}\n", 'cyan') + \
```

Comment:
Curious about the strategy of choosing colors. For consistency between components, it will be great to describe the mappings between colors and entities in the readme somewhere.

Reply:
SGTM